explore/src/main/scala/explore/tabs/ObsTabTiles.scala

```diff
@@ -340,9 +346,13 @@ object ObsTabTiles:
             }
 
           val pa: Option[Angle] =
-            (basicConfiguration.map(_.siteFor), asterismAsNel, vizTime)
-              .mapN((site, asterism, vizTime) =>
-                parallacticAngle(site, asterism.baseTracking, vizTime)
+            PosAngleConstraint.angle
```

Comment:
We could also use the default viz time for the `targetCoords` above, which is used for a bunch of the other tiles. Not sure if that makes sense or not. Currently base coords are used if there isn't a viz time.

Reply:
I think it is better coordinates corrected at the current time than the base coords

Reply:
So, I should change all of them to use the current time if there isn't a viz time?

Reply:
in general I'd say yes, note that there is a tagged type for `CoordinatesAtVizTime` so you know if the coordinates have been corrected.
Now in certain cases the effect of pm correction is quite small and may not matter as much. I think a case are the elevation plots where coordinate changes have little effect on the elevation